stdlib/src/builtin/simd.mojo

```diff
@@ -1483,6 +1483,38 @@ struct SIMD[type: DType, size: Int](
             return rebind[SIMD[target, size]](
                 _f32_to_bfloat16(self.cast[DType.float32]())
             )
+        elif (
+            type.is_integral() and target.is_integral() and target.is_unsigned()
+        ):
+
+            @parameter
+            if target.bitwidth() >= type.bitwidth():
+                alias zero_x1 = Self(0)
+                alias zero_x2 = zero_x1.join(zero_x1)
+                alias zero_x4 = zero_x2.join(zero_x2)
+
+                @parameter
+                if target.bitwidth() // type.bitwidth() == 2:
+                    return bitcast[target, size](zero_x1.interleave(self))
+                elif target.bitwidth() // type.bitwidth() == 4:
+                    return bitcast[target, size](
+                        zero_x2.interleave(zero_x1.interleave(self))
+                    )
+                elif target.bitwidth() // type.bitwidth() == 8:
```

Comment:
In every of theses 3 branches, you are calculating `target.bitwidth() // type.bitwidth()`. Separate variable?